qutebrowser/keyinput/modeparsers.py

```diff
@@ -189,7 +189,7 @@ def _handle_special_key(self, e):
                 return True
             else:
                 return super()._handle_special_key(e)
-        elif config.get('hints', 'mode') != 'number':
+        elif hintmanager.current_mode() != 'number':
```

Comment:
I wish we could use `@property` more, but with PyQt, if any exception happens inside a property handler, you get an `AttributeError` for the property instead, which is really confusing - that's why I'm trying to avoid it.